### DISCUSSION TITLE
How do I display custom shell result?

### DISCUSSION THREAD

**Discussion by YohannesTz** [tobi-wan-kenobi/bumblebee-status#1007] [Q&A]
Hi, I know the shell module exists but was not able to make it work for my needs. (skill issue 😅)

The command I wanted to show the result of looks like this:
```bash
alias cpu_temp="echo \"CPU: \$(sensors | grep 'Core 0' | awk '{print \$3}')\""
```

which I have added to my zshrc. now doing ``cpu_temp`` results the following:
![screenshot-20231216-064532Z-selected](https://github.com/tobi-wan-kenobi/bumblebee-status/assets/52599607/fccc7047-7706-4400-8c2a-b0c27a5b5535)

now in my i3config I have this:
```
bar {
        font pango:Inconsolata 10
        position top
        tray_output none
        status_command bumblebee-status -m memory battery time \
    shell brightness pasink pasource -p time.format="%a %e %b %Y - %I:%M %p" brightness.step="5%" shell.command="cpu_temp" -t solarized-powerline
}
```

which fails to show the result and exit with "cpu_temp exited with code 127"
![screenshot-20231216-064711Z-selected](https://github.com/tobi-wan-kenobi/bumblebee-status/assets/52599607/ba1e7dce-d6a5-463c-a7ae-2ec7d747958d)

I was able to make it work for ``uptime`` but not sure what I am doing wrong here.

any help is appreciated! 🙏

**Comment by tobi-wan-kenobi** [comment]
I do not know exactly which shell Python uses, but I suspect aliases simply don't work.

Can you try creating a shell script instead, make it executable and executing this with its absolute path?
